bitnami/kibana/README.md

```diff
@@ -0,0 +1,310 @@
+| `ingress.tls`                               | Configure TLS for the Kibana Ingress                                                       | `[]`                                                    |
+| `securityContext.enabled`                   | Enable securityContext on for Kibana deployment                                            | `true`                                                  |
+| `securityContext.runAsUser`                 | User for the security context                                                               | `1001`                                                  |
+| `securityContext.fsGroup`                   | Group to configure permissions for volumes                                                  | `1001`                                                  |
+| `securityContext.runAsNonRoot`              | Run containers as non-root users                                                            | `true`                                                  |
+| `resources`                                 | Configure resource requests and limits                                                      | `nil`                                                   |
+| `nodeSelector`                              | Node labels for pod assignment                                                              | `{}`                                                    |
+| `tolerations`                               | Tolerations for pod assignment                                                              | `[]`                                                    |
+| `affinity`                                  | Affinity for pod assignment                                                                 | `{}`                                                    |
+| `podAnnotations`                            | Pod annotations                                                                             | `{}`                                                    |
+| `sidecars`                       | Attach additional containers to the pod                                       | `nil`                                                        |
+| `initContainers`                       | Add additional init containers to the pod                                       | `nil`                                                        |
+| `metrics.enabled`                | Start a side-car prometheus exporter                                          | `false`                                                      |
+| `metrics.service.annotations`                | Prometheus annotations for the Kibana service                                          | `{ prometheus.io/scrape: "true", prometheus.io/port: "80", prometheus.io/path: "_prometheus/metrics" }`                                                      |
```

Comment:
The default port for the Kibana Prometheus exporter seems to be `9563`. See https://github.com/prometheus/prometheus/wiki/Default-port-allocations

Reply:
Copy paste, fixing it